kv/kv.go

```diff
@@ -202,7 +202,7 @@ func (kv *KV) Keys() ([][]byte, error) {
 		it := txn.NewIterator(opts)
 		defer it.Close() //nolint:errcheck
 		for it.Rewind(); it.Valid(); it.Next() {
-			ks = append(ks, it.Item().Key())
+            ks = append(ks, it.Item().KeyCopy(nil))
```

Comment:
could you fix the indentation here

Reply:
Hi @aymanbagabas - this should be sorted:  I've fixed the white space issue and squashed the commits together